root/cdtoc/lookup.tt

```diff
@@ -20,7 +20,7 @@
   [% IF cdstub %]
   <h2>[% l('CD stub found') %]</h2>
   <p>[% l('A CD stub was found that matches the disc ID you provided. If the below tracklist
-           appears correct, you may use it as a starting point as a new MusicBrainz release') %]</p>
+           appears correct, you may use it as a starting point as a new MusicBrainz release.') %]</p>
```

Comment:
Hmm. Do you think it would be better "you may use it as a starting point `for` a new MusicBrainz release" if we're already changing the string?

Reply:
Yes, making an actionable suggestion:

```suggestion
           appears correct, you may use it as a starting point for a new MusicBrainz release.') %]</p>
```